src/generators/web.cr

```diff
@@ -152,8 +150,7 @@ class LuckyCli::Generators::Web
     append_text to: "shard.yml", text: <<-DEPS_LIST
     dependencies:
       lucky:
-        github: luckyframework/lucky
-        version: ~> 0.24.0
+        path: /Users/robert/Documents/repositories/luckyframework/lucky
```

Comment:
🙈 This should be undone.

Reply:
okay but how will you know where I store my code 😆